distribution/scripts/start-client.sh

```diff
@@ -24,4 +24,5 @@ else
   echo "Client is starting up"
 fi
 
-exec /usr/local/bin/npm run start
+export PATH=/usr/bin:/usr/local/bin:$PATH
```

Comment:
FTR: I did this this way to be both compatible with the new and old PATH. (former is the new PATH for `npm` on Centos, the latter was the one on `alpine`).